README.md

```diff
@@ -326,7 +328,7 @@ commands include:
 
 Each of these commands opens the Locust interface for initiating and monitoring the tests, set to [http://localhost:8089](http://localhost:8089).
 Using this interface, you can set the number of users to simulate and their hatch rate, then start and stop the test at
-will. For the host, you can enter a full URL address, or you can simply enter "local", "staging", or "experimental", and
+will. For the host, you can enter a full URL address, or you can simply enter "local", "stg" (for staging), or "exp" (for experimental), and
```

Comment:
If we're not ever going to hit staging, this & other refs to it can probably come out.

Reply:
yeah, it's confusing. I had it beat into me last time that the goal was to test in staging, but is **in the past**, so I'll remove all these refs